### DISCUSSION TITLE
Matching Eclipse RCP version to the correct birt release

### DISCUSSION THREAD

**Discussion by eobrienPilz** [eclipse-birt/birt#1177] [General]
Hi,

I notice in the README there is a useful piece of information "If you want to install BIRT as a plugin, please be aware that BIRT 4.10 requires Eclipse 4.23 and higher".

Is there a page somewhere that states the correct BIRT version to use for any given Eclipse RCP release ?

e.g. our application targets Eclipse 4.16 (2020-06) - JAVA8. I am finding it difficult to find the correct update site to use. I have tried 4.4 and 4.6 but see errors in the target content dialog.

Thanks,
Enda.

**Comment by wimjongman** [comment]
You could be out of luck finding a BIRT version for that Eclipse release. Please use Eclipse 4.25 or 4.26 with BIRT 4.12

You can try the archives to find a matching version. https://eclipse.github.io/birt-website/

**Comment by eobrienPilz** [comment]
Using the link, Its still not clear to me how to match an archived version to the correct eclipse version.

e.g.
	For archived directory M-R1-4.8.0-RC4-201806141605
	I assume this is BIRT version 4.8.
	Does the number at the end (201806141605) represent the eclipse release ?
	
Also, can the BIRT version work with more that one eclipse release ?

**Reply by speckyspooky** [reply]
I try to give you additional information.
The version number is the hint of the building time 201806141605 = YYYYMMDDHH24MIN. This is no hint of the eclipse release number.

I would prefer to you to use the download page of your mentnioned release BIRT 4.8:
https://archive.eclipse.org/birt/downloads/drops/M-R1-4.8.0-RC4-201806141605/

There you can download the **BIRT-all-in-one-designer** with the correct running eclipse platform.
If you have started you can get detailed information at the help menu -> About Eclipse Platform.

There you get many detailed information. The eclipse version of BIRT 4.8 => Version: 4.8.0.v20180611-0656

**BIRT-eclipse version screen (BIRT 4.8)**
![BIRT-eclipse-platform](https://user-images.githubusercontent.com/41593722/211201379-80d80589-c212-48c4-8380-e1aecf070f21.png)

**Comment by eobrienPilz** [comment]
I downloaded 4.8, 4.9, 4.10 and 4.11. This is what I see in about.

Birt 4.8 - Eclipse 4.8
Birt 4.9 - Eclipse 4.9
Birt 4.10 - Eclipse 4.25
Birt 4.11 - Eclipse 4.25

There seems to be a big gap in Eclipse versions from 4.9 to 4.25. 
So, back to my other question, can a BIRT version work with more that one eclipse release ?
i.e. 
BIRT 4.9 - Eclipse 4.16

**Comment by wimjongman** [comment]
We don't have data on that. You will have to try that yourself.
